pytest_fixtures/component/taxonomy.py

```diff
@@ -148,14 +123,25 @@ def function_sca_manifest_org(function_org, function_sca_manifest, target_sat):
     return function_org
 
 
+@pytest.fixture
+def function_els_sca_manifest_org(function_org, function_sca_els_manifest, target_sat):
+    """Creates an organization and uploads an SCA mode manifest generated with manifester"""
+    target_sat.upload_manifest(function_org.id, function_sca_els_manifest.content)
+    return function_org
+
+
 @pytest.fixture(scope='module')
-def module_gt_manifest_org(module_target_sat):
-    """Creates a new org and loads GT manifest in the new org"""
-    org = module_target_sat.api.Organization().create()
-    manifest = clone(org_environment_access=True, name='golden_ticket')
-    module_target_sat.upload_manifest(org.id, manifest.content, interface='CLI')
-    org.manifest_filename = manifest.filename
-    return org
+def module_els_sca_manifest_org(module_org, module_sca_els_manifest, module_target_sat):
+    """Creates an organization and uploads an SCA mode manifest generated with manifester"""
+    module_target_sat.upload_manifest(module_org.id, module_sca_els_manifest.content)
+    return module_org
+
+
+@pytest.fixture(scope='class')
+def class_els_sca_manifest_org(class_org, class_sca_els_manifest, class_target_sat):
+    """Creates an organization and uploads an SCA mode manifest generated with manifester"""
+    class_target_sat.upload_manifest(class_org.id, class_sca_els_manifest.content)
+    return class_org
```

Comment:
This seems to be extra too. I was not sure about it during the first review, but now I checked more thoroughly.

Reply:
Ack, thank you. I had a merge conflict on this file that I may have resolved incorrectly. I will revisit it.